spec/models/grda_warehouse/youth/youth_intake_permissions_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
```

Comment:
@awisema, can you add a test that shows that a client with an ongoing youth intake at an agency that isn't the current user doesn't prevent the current user from starting a new youth intake?  Also, probably worth checking that a user at the agency where there *is* an ongoing intake can't create a second.